gym/wrappers/gray_scale_observation.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import cv2
```

Comment:
opencv is not a direct dependency, rather, it is installed when atari_py is installed; so I think we should move opencv into the class definition (otherwise, gym import / wrapper import will fail when opencv is not installed) (kinda like in this commit: https://github.com/openai/gym/commit/98917cf5d3a4808de65254bd8fdf380b0f459a53)

Reply:
It's adjusted accordingly to 98917cf